karafka.gemspec

```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'bundler', '~> 1.2'
 
   spec.add_dependency 'ruby-kafka', '= 0.3.17'
-  spec.add_dependency 'sidekiq', '~> 4.2'
+  spec.add_dependency 'sidekiq', '~> 5.0'
```

Comment:
@maximebedard  done :) sorry for any inconvenience  that changed caused and thank you for an inspiration for another Karafka feature!